maro/simulator/scenarios/vm_scheduling/topologies/azure.2019.10k/config.yml

```diff
@@ -23,22 +23,44 @@ MAX_MEM_OVERSUBSCRIPTION_RATE: 1
 # Max CPU utilization rate.
 MAX_UTILIZATION_RATE: 1
 
-PM:
-  - PM_type: 0 # PM type is currently "int" only.
-    amount: 100
-    CPU: 32
-  # GPU: 0
-    memory: 128
-    power_curve:
-      calibration_parameter: 1.4
-      busy_power: 10
-      idle_power: 1
-# - PM_type: 1
-#   amount: 50
-#   CPU: 32
-#   GPU: 0
-#   memory: 128
-#   power_curve:
-#     calibration_parameter: 1.4
-#     busy_power: 10
-#     idle_power: 1
+components:
+  pm:
+    - pm_type: 0
+      cpu: 32
+      memory: 128
+      power_curve:
+        calibration_parameter: 1.4
+        busy_power: 10
+        idle_power: 1
+    - pm_type: 1
+      cpu: 16
+      memory: 112
+      power_curve:
+        calibration_parameter: 1.4
+        busy_power: 10
+        idle_power: 1
+  rack:
+    - type: 'JP1-a'
+      pm:
+        - pm_type: 0
+          amount: 50
+    - type: 'JP1-b'
+      pm:
+        - pm_type: 1
+          amount: 50
+  cluster:
+    - type: 'JP1'
+      rack:
+        - rack_type: 'JP1-a'
+        - rack_type: 'JP1-b'
+
+architecture:
+  region:
+    - name: 'APAC'
+      zone:
+        - name: 'asia-northeast1'
+          data_center:
+            - name: 'Japan'
+              cluster:
+                - type: 'JP1'
+                  rack_amount: 2
```

Comment:
cluster_amount?

Reply:
`rack_amount` means the total rack number under `'JP1'` cluster

Reply:
So in this situation, there are two racks under `'JP1'` cluster: `'JP1-a'` and `'JP1-b'`